src/ArcherContainer.js

```diff
@@ -254,7 +254,12 @@ export class ArcherContainer extends React.Component<Props, State> {
         parentCoordinates,
       );
       const markerId = this.getMarkerId(from, to);
-      const clickHandler = this.props.onClick;
+      const clickHandler = this.props.onClick
+        ? (evt) => {
+          if (this.props.onClick) {
+            this.props.onClick(evt, markerId, from.id, to.id);
```

Comment:
@pierpo could also pass an object like `arrowData { marker, from, to }` or similar ... and perhaps this same event should also apply to the marker